service-base/src/main/java/org/eclipse/hono/service/management/credentials/PasswordSecret.java

```diff
@@ -94,18 +128,20 @@ public void checkValidity() {
     }
 
     /**
-     * Encodes a clear text password contained in the <em>pwd-plain</em> field.
+     * Encodes the clear text password contained in the <em>pwd-plain</em> field.
```

Comment:
there is no *pwd-plain* field in this class. I guess you mean the *passwordPlain* field.

Reply:
yes sure.. this is the `pwd-plain` property defined in the API spec